en/docs/administer/product-security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md

```diff
@@ -141,67 +141,6 @@ This tool is packaged with WSO2 API Manager by default. Follow the steps below t
 This tool can run standalone and therefore cater to multiple products. This means that if you are using multiple WSO2 products and need to delete the user's identity from all products at once, you can do so by running the tool in standalone mode.
 For information on how to build and run the Forget-Me tool, see [Removing References to Deleted User Identities in WSO2 Products](https://docs.wso2.com/display/ADMIN44x/Removing+References+to+Deleted+User+Identities+in+WSO2+Products) in the WSO2 Administration Guide.
```

Comment:
The doc links should not point to the admin44x space anymore.

Reply:
This section is not relevant to my PR. This section(the commented one) was not changed by PR. I have fixed my PR with the sections that are related to my change with 6c779f9. You may change the highlighted section.